src/util/embedTagHelpers.tsx

```diff
@@ -95,6 +118,21 @@ export const parseEmbedTag = (embedTag?: string): Embed | undefined => {
   return obj as unknown as Embed;
 };
 
+const attributeRegex = /[A-Z]/g;
+
+export const createEmbedTagV2 = (data: Record<string, any>) => {
```

Comment:
Hadde det gitt verdi å lage tester for dette?

Reply:
Det skal jeg nok klare å få til.